helpers/controllers/authController.ts

```diff
@@ -93,6 +92,10 @@ export const logout = async (_parent: void, _: void, ctx: Context) => {
           error: err.message
         })
       }
+      res.setHeader(
+        'Set-Cookie',
+        `connect.sid=false; expires=${new Date(Date.now())}; Path=/`
+      )
```

Comment:
According to [this StackOverflow question](https://stackoverflow.com/q/5285940), the correct way to delete the cookie server-side is to set an expiration date in the past, like `Thu, 01 Jan 1970 00:00:00 GMT` to be 100% sure the cookie will be deleted. The cookie value can also be empty.

```suggestion
      res.setHeader(
        'Set-Cookie',
        `connect.sid=; expires=${new Date(0).toUTCString()}; Path=/`
      )
```

Also just FYI, `new Date()` already creates a date object with the current timestamp, so doing `new Date(Date.now())` is redundant ;)

Reply:
Thanks. Fixed.